server/mdm/windows/windows_mdm.go

```diff
@@ -0,0 +1,12 @@
+package windows_mdm
+
+import "github.com/fleetdm/fleet/v4/server/mdm/internal/commonmdm"
+
+const (
+	// DiscoveryPath is Fleet's HTTP path for the Windows MDM Discovery endpoint.
+	DiscoveryPath = "/EnrollmentServer/Discovery.svc"
```

Comment:
Is that correct, i.e. is it always that path? I took it from the ticket's description but not sure if it was just an example of a possible path, or THE path :)

Reply:
I think we want to have **THE** path :D, whether or not it's `/EnrollmentServer/Discovery.svc` probably a question for @marcosd4h and @rachaelshaw (API DRI)

IMO we want something that works better with the rest of the API, but I don't know if there are any windows requirements.

Reply:
Probably a good idea to document this path in the API for contributors as well

Reply:
that looks way better!! just a reminder to double check with Rachel in your PR

Reply:
Sounds good, so as mentioned I'll leave the path as-is for this PR and it will be updated in Marcos'. I won't document it in  API for contributors in this PR given that it will change.